### PR TITLE
feat(ui): improved the UI for announcement page

### LINKED ISSUE #3524: [ui] : revamp the announcement form UI.
**Is your feature request related to a problem? Please describe.**
We need to improve the UI for announcement page. 
| Screenshot from 2023-01-25 16-52-08 |
|-|
|Announcement Page|

**Describe the solution you'd like**
Suggest a design and before raising a PR, kindly post the UI in `#ux` channel in slack so that we can provide feedback. 

**Additional context**
The annoucement page requires `admin` access, if you want to reproduce the bug you can remove [this](https://github.com/CircuitVerse/CircuitVerse/blob/32b382f1b0caf2a214d8a573078d19502d1a8ead/app/controllers/announcements_controller.rb#L5) line. Remember to not to add this in commit.

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by niladrix719**
Fixes #3524 

#### Describe the changes you have made in this PR -

I have improved the UI of the Announcement Page

### Screenshots of the changes (If any) -

Screenshot from 2023-07-26 17-27-48

### CODE REVIEW

**Review by VaibhavUpreti — APPROVED** [review]
cc @vedant-jain03

**Comment by niladrix719** [comment]
Hi @tanmoysrt is it a issue from my end ? i am getting : `This GitHub token doesn't have write permission of Review API` in Ruby on Rails CI / test

**Comment by tanmoysrt** [comment]
Reviewdog is reporting some issues. Can you check the `Files Changed` tab? Normally if there is any issue, those are listed on that page normally.

**Comment by niladrix719** [comment]
@tanmoysrt the files changed tab looks fine tho

**Review by tanmoysrt — APPROVED** [review]
Looks good :+1

File: app/views/announcements/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <div class="container">
   <div class="row center-row">
-    <div class="col-xs col-sm col-md col-lg">
+    <div class="col-xs col-sm col-md col-lg an-lg-green">
```
Comment (tanmoysrt) [inline app/views/announcements/index.html.erb:3]:
```suggestion
    <div class="col-xs col-sm col-md col-lg announcement-container">
```

We can keep the class name as `announcement-container`

File: app/assets/stylesheets/announcements.scss [hunk 2]
```diff
@@ -23,6 +23,36 @@
...
+}
+
+.announcement-box {
+  background-color: $white;
+}
+
+.an-lg-green {
+  background-color: $card-green;
+}
+
```
Comment (tanmoysrt) [inline app/assets/stylesheets/announcements.scss:55]:
```suggestion
.announcement-container {
  background-color: $card-green;
}

```

Keeping it `announcement-container` will be better ig

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: app/views/announcements/index.html.erb (hunk 1 above)
Comment (tanmoysrt) [inline app/views/announcements/index.html.erb:3]: Provide a meaningful class name instead of `an-lg-green`
ig reviewdog maybe failing for this reason

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: app/views/announcements/index.html.erb (hunk 1 above)
Comment (niladrix719) [reply app/views/announcements/index.html.erb]: some things wrong with this PR i am opening a new one #4288

**Pull request closed without merging**